server/tests/database/test_notification_job_data_handler.py

```
@@ -0,0 +1,221 @@
+def test_get_jobs_by_target_exclude_other_org(app):
+    make_job(scheduledAt=datetime.now(timezone.utc) + timedelta(hours=1))
+    make_job(scheduledAt=datetime.now(timezone.utc) + timedelta(hours=2))
+    make_job(orgId=2, scheduledAt=datetime.now(timezone.utc) + timedelta(hours=2))
+
+    jobs = NotificationJobDataHandler.get_jobs_by_target(ORG_ID, TARGET_ID, JOB_TYPE)
+
+    assert len(jobs) == 2
+    assert all(j.targetId == TARGET_ID for j in jobs)
+    assert all(j.type == JOB_TYPE for j in jobs)
+
+def test_get_jobs_by_target_wrong_type_returns_empty(app):
```

Ruff (0.15.20)

[warning] Unused function argument: `app`

(ARG001)